backend/src/docs/v1.api-spec.yaml

```diff
@@ -109,12 +149,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  /appConfigForm:
+  /webade/appConfigForm:
```

Comment:
If we're adopting `/webade/{webAdeEnv}`, we should also consider doing the same to the API shape for the POST appConfigForm endpoint as well. Both the `*ConfigForm` actions are currently using a required ClientEnvironment body parameter for it so it may be worth embedding it into the path.

Reply:
Yes I think that's worth looking at, need to give it some more thought. Will take it up after this PR, but leave the POST ones as they are for this scope.